Doc/Tutorial/chapter_cluster.tex

```diff
@@ -202,11 +202,7 @@ \subsection*{Calculating the distance matrix}
 \end{minted}
 which can be rewritten as
 \begin{minted}{python}
-[array([], dtype=float64),
- array([ 16.]),
- array([ 64.,  16.]),
- array([  1.,   9.,  49.])
-]
+[array([], dtype=float64), array([16.0]), array([64.0, 16.0]), array([1.0, 9.0, 49.0])]
```

Comment:
This change is arguably less readable - all in one line for a (triangular) matrix